src/main/java/seedu/address/logic/commands/issue/CloseIssueCommand.java

```diff
@@ -29,8 +31,15 @@
 
     public static final String MESSAGE_CLOSE_ISSUE_CLOSED = "Issue has already been closed.";
 
+    private final Logger logger = LogsCenter.getLogger(CloseIssueCommand.class);
+
     private final Index targetIndex;
 
+    /**
```

Comment:
Missing punctuation and @param description should be "Target index of the issue in the filtered issue list to close." (separated from the param name)